packages/core/src/lib/readDocumentation.js

```diff
@@ -54,6 +54,14 @@ module.exports = function(pattern, patternlab) {
       if (markdownObject.links) {
         pattern.links = markdownObject.links;
       }
+
+      if (
+        markdownObject.hasOwnProperty('subfolder') &&
+        !markdownObject.subfolder
```

Comment:
i tested this exhaustively with the commented `subbed` key until finally finding this - oops!